templates/baremetalset/cloudinit/userdata

```diff
@@ -15,3 +15,6 @@ chpasswd:
 bootcmd:
   # fix BLS entries
   - set -x; if [ -e /boot/loader/entries/ffffffffffffffffffffffffffffffff-* ]; then MACHINEID=$(cat /etc/machine-id) && rename "ffffffffffffffffffffffffffffffff" "$MACHINEID" /boot/loader/entries/ffffffffffffffffffffffffffffffff-* ; fi
+runcmd:
+  # grow volumes on hardened image if command exists
+  - set -x; if [ -e /usr/local/sbin/growvols ]; then /usr/local/sbin/growvols --yes {{ range .GrowvolsArgs}}{{.|trim}} {{end}}; fi
```

Comment:
from where is  `/usr/local/sbin/growvols` coming? is it part of the overcloud image?

Reply:
yes

Reply:
only the overcloud images with lvm. Logic comes from  https://github.com/openstack/tripleo-ansible/blob/stable/wallaby/tripleo_ansible/playbooks/cli-overcloud-node-growvols.yaml which is run on all nodes when deploying with python-tripleoclient